packages/kit/build.config.ts

```diff
@@ -7,6 +7,7 @@ export default defineBuildConfig({
   ],
   externals: [
     '@nuxt/schema',
+    'nuxt/schema',
```

Comment:
we need to use `@nuxt/schema` within Kit as there is no `nuxt` dependency. The end user is the only one who should use `nuxt/schema` as a type import/augmentation...

Reply:
Yes, I understand. We need to find a different solution than using `nuxt/schema` with `@nuxt/kit`...